release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1341,12 +1341,14 @@ For any {product-title} release, always review the instructions on xref:../updat
 ====
 
 [id="ocp-4-7-0-ga"]
-=== RHBA-2021:1234 - {product-title} 4.7 image release and bug fix advisory
+=== RHEA-2020:5633 - {product-title} 4.7.0 image release, bug fix, and security update advisory
 
-Issued: 2021-xx-xx
+Issued: 2021-02-24
 
-{product-title} release 4.7 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2020:1234[RHBA-2020:1234] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:5678[RHBA-2020:5678] advisory.
+{product-title} release 4.7.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHEA-2020:5633[RHEA-2020:5633] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHEA-2020:5634[RHEA-2020:5634] advisory.
+
+This release also includes security updates. The list of security updates is documented in the link:https://access.redhat.com/errata/RHSA-2020:5635[RHSA-2020:5635] advisory.
```

Comment:
The second sentence in this paragraph could be minimised as follows:

"The security updates are listed in the link:https://access.redhat.com/errata/RHSA-2020:5635[RHSA-2020:5635] advisory."

Reply:
As per the previous comment regarding URLs, I am assuming this link is not yet active.

Reply:
Actually removing this line entirely as now the images and RPMs advisories are also RHSA, so it fits in our usual paradigm :)